packages/flutter/test/widgets/gesture_detector_test.dart

```diff
@@ -249,10 +248,8 @@ void main() {
     );
 
     final RenderSemanticsGestureHandler renderObj2 = tester.renderObject(find.byType(GestureDetector));
-    final GestureTapCallback actualCallback2 = renderObj2.onTap;
 
     expect(renderObj1, same(renderObj2));
-    expect(actualCallback1, same(actualCallback2)); // Should be cached.
   });
 
```

Comment:
can you elaborate on why this test has to change?

Reply:
Looks to me like this would regress https://github.com/flutter/flutter/issues/11552. We don't want to do that, that would be a performance problem.

Reply:
You can probably solve this by using methods rather than closures for the callbacks set in the default delegate.

Reply:
Yes, it's in the PR description "Regarding the changed test"

Reply:
if we've decided not to maintain this invariant, we should just remove the test. Changing the test likes this seems to just make the test redundant with much more low-level tests. Do you have numbers showing that the performance change is not an issue?